packages/EasyStandard/src/Sniffs/Arrays/OrderArrayKeysAlphabeticallySniff.php

```diff
@@ -0,0 +1,255 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyStandard\Sniffs\Arrays;
+
+use Error;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PhpParser\Comment;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Expr\ArrayItem;
+use PhpParser\ParserFactory;
+use PhpParser\PrettyPrinter\Standard;
+use SlevomatCodingStandard\Helpers\TokenHelper;
+
+final class OrderArrayKeysAlphabeticallySniff implements Sniff
+{
+    /**
+     * @var string
+     */
+    public const ARRAY_KEYS_SORT_ALPHABETICALLY = 'ArrayKeysSortAlphabetically';
+
+    /**
+     * @var string
+     */
+    private const COMMENT_CONTENT = '[comment]';
+
+    /**
+     * @var mixed[]
+     */
+    public $skipPatterns = [
+        T_FUNCTION => ['/^provide.*/'],
```

Comment:
Let's not use the default pattern here. It's a decision made only for our projects. We will add `/^provide.*` on a per-project basis in the configs.
Don't forget to adjust the docs.

Reply:
Done.